src/core/store/notifications/mutations.js

```diff
@@ -20,12 +20,19 @@ const INIT_STORE = function (state) {
 const ADD_NOTIFICATION = function (state, obj) {
   delete obj.notification;
   state.notifications.push(obj);
```

Comment:
should this be:  
```
if (state.notifications.length >=20) {
  state.notifications.shift();
}
state.notifications.push(obj);
```

Reply:
YES LOL i don't know why i added the push twice aha will remove now